auth_pass_ip_restriction/readme/USAGE.rst

```diff
@@ -0,0 +1,9 @@
+When this module is installed, you can enable the regular password login form by
+adding `?debug=1` to your browser's url.
+
+Configuring the IP whitelist is done in the `odoo.conf` configuration file, with
+the `auth_pass_ip_whitelist` parameter like so:
+```
+auth_pass_ip_whitelist = ["127.0.0.1", "192.168.1.7"]
+```
+If this parameter is not set, it defaults to `["127.0.0.1"]`.
```

Comment:
We need to add here some documentation of our new config parameter for whitelisting some URL's, plus an example of for which URL's this could be used (eg: auth_saml)